documentation/modules/exploit/multi/misc/consul_rexec_exec.md

```diff
@@ -0,0 +1,105 @@
+## Vulnerable Application
+
```

Comment:
List the vulnerable application and versions affected here.

Reply:
See https://github.com/rapid7/metasploit-framework/pull/10444/commits/53a3caba2b64f8cb16b3e230a9f63fe5647edec9